### PR TITLE
Wavelengths by Hand

### PR DESCRIPTION
**Pull request by profxj**
Allows the user to input a set of pixel, wavelength pairs to 
guid the wavelength solution.  

Tested on Kastr and LRISr

### CODE REVIEW

File: src/pypit.py [hunk 1]
```diff
@@ -12,7 +12,8 @@
 # Import PYPIT routines
 import ardebug
 debug = ardebug.init()
-#debug['develop'] = True
+debug['develop'] = True
```
Comment (rcooke-ast) [inline src/pypit.py]: This should be commented out when pushing to GitHub (not all users will want the develop setting on).

File: src/arload.py [hunk 2]
```diff
@@ -43,6 +43,8 @@ def argflag_init():
                 'extract':dict({'binby':1.0}),
                 'load':dict({'extracted':False, 'calibrated':False}),
                 'calibrate':dict({'cwpolyorder':2, 'threshold':3.0, 'polyorderpri':4, 'polyordersec':8, 'pcapri':[4,2,2,0], 'pcasec':[5,4,3,2,1,1], 'detection':6.0, 'method':'simple', 'nfitpix':7, 'idfile':'wave_ThAr_3100-11000.npy', 'linelist':'arclist.ThAr', 'numsearch':20, 'sigmacut':2.0}),
```
Comment (rcooke-ast) [inline src/arload.py]: I would have expected `id_pix` and `id_wave` to go in the calibrate dict, above. Then, the input file will need the following lines to set the pixels and wavelengths.

arc calibrate id_pix 220.63808792,417.17122925,562.95948074,669.07189712,884.38191292
arc calibrate id_wave 5868.4165,6334.4276,6678.2766,6929.4672,7438.8981

File: src/pypit.py (hunk 1 above)
Comment (profxj) [inline src/pypit.py]: Agreed

File: src/arload.py (hunk 2 above)
Comment (profxj) [inline src/arload.py]: Yup.  That is best.
Modified.

**Comment by profxj** [comment]
Passed TEST suite.

Merging

**Pull request merged**